draft-pardue-moq-qlog-moq-events.md

```diff
@@ -254,11 +250,13 @@ is created. It has Core importance level.
 MOQTObjectDatagramCreated = {
     track_alias: uint64
     group_id: uint64
-    object_id: uint64
+    ? object_id: uint64
     publisher_priority: uint8
-    extension_headers_length: uint64
+    ? extension_headers_length: uint64
     ? extension_headers: [* MOQTExtensionHeader]
     ? object_payload: RawInfo
+    ? object_status: uint64
+    end_of_group: bool
```

Comment:
Should we just add the type field to the datagram objects? I think that could make it easier to understand why certain fields are present, and we don't need the end_of_group bool.